hammer_cli_foreman_discovery.gemspec

```diff
@@ -16,11 +16,10 @@ Gem::Specification.new do |s|
   s.description = <<DESC
 Contains the code for managing host discovery in foreman(results and progress) in the Hammer CLI.
 DESC
-
   s.files = Dir['{lib,locale,config}/**/*', 'LICENSE', 'README*']
   s.extra_rdoc_files = Dir['LICENSE', 'README*']
   s.require_paths = ["lib"]
 
-  s.add_dependency "hammer_cli_foreman", ">= 0.1.2"
+  s.add_dependency "hammer_cli_foreman", "> 0.1.3"
```

Comment:
0.1.3 is the latest release, by the way, so I don't think you should release this gem until 0.1.4 is available.